tests/gpu/linalg_test/linalg_test.cpp

```diff
@@ -25,6 +25,7 @@ TEST_F(linalg_Test, gpu_BkUt_Svd_truncate2) {
   auto con_T2 = Contract(Contract(res[1], res[0]), res[2]);
 }
 
+/*
 TEST_F(linalg_Test, gpu_BkUt_Svd_truncate3) {
   std::vector<UniTensor> res = linalg::Svd_truncate(svd_T, 200, 0, true);
```

Comment:
Maybe just skip them or disable them, so do we know we have to fix them in the future.

Reply:
This problem occurs since the answer from dense is wrong. I think gpu_BkUt_Svd_truncate1 is enough.